src/Symfony/Component/Intl/Data/Generator/CurrencyDataGenerator.php

```diff
@@ -137,7 +137,9 @@ protected function generateDataForMeta(BundleEntryReaderInterface $reader, $temp
      */
     private function generateSymbolNamePairs(ArrayAccessibleResourceBundle $rootBundle)
     {
-        $symbolNamePairs = iterator_to_array($rootBundle['Currencies']);
+        $symbolNamePairs = array_map(function ($pair) {
+            return array_slice(iterator_to_array($pair), 0, 2);
+        }, iterator_to_array($rootBundle['Currencies']));
```

Comment:
We need to do this so that php generator doesn't dump the following invalid code:

```php
        'ESP' => [
            0 => '₧',
            1 => 'pesseta espanyola',
            2 => 
            ResourceBundle::__set_state(array(
            )],
        ],
```

Json generator used to simply dump `{}`.

We never use the third index. It's hardly ever present. If we dumped it it would like like this:


```php
        'ESP' => [
            0 => '₧',
            1 => 'pesseta espanyola',
            2 => [
                0 => '¤ #,##0.00',
                1 => ',',
                2 => '.',
            ],
        ],
```